doc/nrf/protocols/matter/getting_started/tools.rst

```diff
@@ -164,132 +164,125 @@ For more information about the ZAP tool, see the official `ZCL Advanced Platform
 
-   * Case 2: If your currently installed ZAP version does not match the recommended one or no ZAP version is installed on your device, you will see a message similar to the following one:
+      After the installation is done, you can close the ZAP tool GUI window.
 
-     .. code-block::
+      To read more about Matter west commands, see the :ref:`ug_matter_gs_tools_matter_west_commands` section.
 
-        Your currently installed ZAP tool version 2022.12.19 does not match the recommended one: 2022.12.20.
+   .. tab:: Manual installation
 
-     Alternatively, this message can look like the following one:
+      You can manually install the ZAP tool using one of the following methods:
 
-     .. code-block::
+      * Download the ZAP tool package in a compatible version manually from the Assets section in `ZCL Advanced Platform releases`_.
+      * Configure the tool and manually compile it using the instructions in the official `ZCL Advanced Platform`_ documentation.
 
-        No ZAP tool version was found installed on this device.
+      Both of these methods require you to add the ZAP tool location to the system :envvar:`PATH` environment variables.
 
-     In this case, the package download process will start automatically:
+..
 
-     .. parsed-literal::
-        :class: highlight
+.. _ug_matter_gs_tools_matter_west_commands:
 
-        Trying to download ZAP tool package matching your system and recommended version.
-        100% [......................................................................] 150136551 / 150136551
-        ZAP tool package was downloaded and extracted in the given location.
+Matter west commands
+********************
 
-     Depending on your operating system, the download will conclude with the following message, with updated *zap_cli_location* and *zap_location* paths, which you will need in the following step:
+Matter west commands are a set of commands dedicated for Matter-related purposes and integrated with :ref:`zephyr:west`.
+You can use them to simplify work with the Matter application project.
 
-     .. tabs::
+.. _ug_matter_gs_tools_matter_west_commands_zap_tool:
 
-        .. group-tab:: Windows
+ZAP tool west commands
+======================
 
-           .. parsed-literal::
-              :class: highlight
+To simplify work with the :ref:`ug_matter_gs_tools_zap`, there are two commands implemented within the Matter west commands set:
 
-              #######################################################################################
-              # Please add the following location(s) to the system PATH:                            #
-              # *zap_location*                                                                      #
-              #######################################################################################
+* ``zap-gui``
+* ``zap-generate``
 
```

Comment:
```suggestion
By default, both commands look for a :file:`.zap` file in the current directory.
You can also specify the path to the :file:`.zap` file explicitly.
```
Moving this information here, so it doesn't need to be at the start of both commands as well as in the listed arguments.